email-question-draft.md

```diff
@@ -0,0 +1,36 @@
+to: .NET Foundation Members
+subject: .NET Foundation Board Elections - Your Turn to Ask the Questions!
+---
+
+> "Asking questions is the first way to begin change." - Kubra Sait
+
+Foundation Members,
+
+As you're likely already aware, the [Foundation Board of Directors campaign is currently underway](https://election.dotnetfoundation.org), with the election scheduled to take place March 21-28.  
+
+But while we may have [candidate statements available to peruse](https://election.dotnetfoundation.org/candidates), we were missing one of the most important pieces of the process -- **questions from you!**
+
+## Have a Question? Now's Your Chance to Ask!
+
+The candidates are ready to [hear your questions](https://election.dotnetfoundation.org/questions). Here's how it works:
+
+* Create [a new issue in the election GitHub repository](https://github.com/dotnet-foundation/election/issues/new?labels=candidate-question&template=electionquestion.md)
+* We'll auto-label the question `candidate-question`.
+* We'll mention the list of candidates so that they're aware the question has been asked
+* The candidates can respond and answer on the GitHub issue if they choose to do so (and we'll try to keep the responses to just their answers, to keep things tidy).
+* The issues will remain [available on the GitHub repo](https://github.com/dotnet-foundation/election/labels/candidate-question) through the election so anyone can browse them.
+
+The .NET Foundation supports *_you_*. If you want to know where a board member stands on something, or how they would approach a situation, or more about their commitment to a topic you care about, don't miss this opportunity to make your voice heard!
+
+## Also -- There's Still Time to Become a Foundation Member
+
+Do you know someone who you think cares about the future of .NET, and is involved in the .NET community in some way? Someone who teaches, or writes, or codes, or contributes in the .NET space? Tell them to join already! Even beyond the election, there will be plenty of great ways to connect and contribute.
+
+Things to remember:
+
+* While there is a review process, it's not designed to be exclusionary. We're not pouring over your life's work. We're looking for folks who have demonstrated that they have a committment to bringing .NET to others and being active participants in the community. If you're created an wildly popular open-source project, that's great! -- it's just not a barrier for entry if you haven't.
+* Same thing with the dues. The Foundation is asking for $100 in dues, so that we will have a budget to do some great things. But please, do not let the dues stop you! The dues are suggested; if that's not where you can allocate funds now, we don't want that to stop you from joining. What we care about most is a willingness to help out and a commitment to see .NET and OSS succeed. We'll be working on ways of making this better in the future, shortly after we, uh, become an official "we".
```

Comment:
```suggestion
* Same thing with the dues. The Foundation is asking for $100 in dues, so that we can have budget to do some great things. But please, do not let the dues stop you! The dues are suggested; if that's not where you can allocate funds now, we don't want that to stop you from joining. What we care about the  most is a willingness to help out and a commitment to see .NET and OSS succeed. We'll be working on ways of making this better in the future, shortly after we, uh, become an official "we".
```